src/controllers/tripsController.js

```
@@ -75,6 +78,7 @@ class tripsController {
         }
       });
     } catch (error) {
+      console.log(error);
```

Better to push console.log free-code
```suggestion
      
      
```

---

You mean an empty catch block?

---

Nope. You can still pass res and req in this method so that you could be able to return meaningful message to the user